### PR TITLE
Feature flexberry logging

### PR DESCRIPTION
**Pull request by kafnevod**
Add @module ember-flexberry header @yuidoc 

Indent remain 2spaces as in other modules 
(indent 1space cause error: Expect indendation of 2 characters at...)

### CODE REVIEW

File: addon/controllers/i-i-s-caseberry-logging-objects-application-log-l.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+ * @module ember-flexberry
+ */
+import ListFormController from 'ember-flexberry/controllers/list-form';
```
Comment (AlexanderElyseev) [inline addon/controllers/i-i-s-caseberry-logging-objects-application-log-l.js]: Short reexport syntax is much better: 
`export { default } from 'ember-flexberry/controllers/list-form';`

**Comment by AlexanderElyseev** [comment]
1. Add all [yuidoc comments](http://yui.github.io/yuidoc/syntax/) (class, method, property).
2. Add unit-tests.

File: package.json [hunk 2]
```diff
@@ -40,6 +40,7 @@
     "ember-disable-prototype-extensions": "^1.0.0",
     "ember-disable-proxy-controllers": "^1.0.1",
     "ember-export-application-global": "^1.0.4",
+    "ember-link-action": "0.0.34",
```
Comment (ehaberev) [inline package.json]: Unnecessary dependency

File: package.json [hunk 3]
```diff
@@ -57,6 +58,7 @@
     "ember-local-storage": "0.0.10",
     "ember-simple-auth": "1.0.1",
     "ember-truth-helpers": "1.2.0",
+    "ember-cli-htmlbars": "^1.0.1",
```
Comment (ehaberev) [inline package.json]: Unnecessary dependency

File: addon/services/flexberry-logging.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+      processId:document.location.href,
+      processName:'EMBER-FLEXBERRY',
+      threadName:'',
+      win32ThreadId:'',
+      message:message,
+      formattedMessage:formattedMessage
+    };
+    let logModel = 'i-i-s-caseberry-logging-objects-application-log';	//Model for applicationLog
+    if (this.get('flexberryStore').peekAll(logModel).findBy('message', message) !== undefined) {	//This message exists in the store?
+      //alert("RepeatedRecord="+message);
```
Comment (ehaberev) [inline addon/services/flexberry-logging.js]: Redundant code

File: addon/services/flexberry-logging.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+      return;	//return to avoid infinity loop when message is genetaried in save/ajax stage
+    }
+
+    //alert("SendMessage ="+message);
```
Comment (ehaberev) [inline addon/services/flexberry-logging.js]: Redundant code

**Pull request merged**